### PR TITLE
SonarQube rebuild

### PR DESCRIPTION
**Pull request by dglynn**
* Fork geerlinguys role and make some amendments due to ESearch now
    can't run as root. SonarQube installed and tested with GItHub login.

### CODE REVIEW

**Comment by dglynn** [comment]
I used the geerlingguy role as a base and this other [role](https://github.com/lrk/ansible-role-sonarqube) to get this working with the amendments.

File: tasks/main.yml [hunk 1]
```diff
@@ -46,21 +62,40 @@
     state: link
   when: "ansible_service_mgr != 'systemd'"
 
+- name: Set RUN_AS user to sonar, to avoid ElasticSearch failing with 'cannot run as root'
+  lineinfile:
+    path: /usr/local/sonar/bin/linux-x86-64/sonar.sh
+    firstmatch: yes
+    regexp: '^#RUN_AS_USER='
```
Comment (pppontusw) [inline tasks/main.yml]: This doesn't make sense to me, why don't you remove `firstmatch: yes` and instead set regexp to simply `'RUN_AS_USER'` as that should match the line regardless if it's commented or not and replace it with the correct line

File: tasks/main.yml [hunk 2]
```diff
@@ -46,21 +62,40 @@
...
+    enabled: yes
+
+- name: "Ensure SonarQube files permissions"
+  file:
+    path: /usr/local/sonar
+    state: directory
+    recurse: yes
+    owner: "{{ sonar_user }}"
+    group: "{{ sonar_group }}"
+    mode: "go-w"
```
Comment (pppontusw) [inline tasks/main.yml]: This is more a point for discussion in the team but I would definitely prefer "number" mode statements.. `go-w` doesn't make much sense to me 🔸

**Review by pppontusw — CHANGES_REQUESTED** [review]

File: tasks/main.yml (hunk 2 above)
Comment (dglynn) [reply tasks/main.yml]: yeah, this is true, it was a copy pasta from the second role from me. I will amend.

File: tasks/main.yml (hunk 1 above)
Comment (dglynn) [reply tasks/main.yml]: ok can do. I was trying to figure out how to make ansible report it changed the line once. If you run the playbook, it always says changed but it doesn't actually change it after the first run. I was using firstmatch after reading more on the module.

**Review by pppontusw — APPROVED** [review]

**Pull request merged**